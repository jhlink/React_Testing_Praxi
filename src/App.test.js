import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
});


it('should render label', () => {
  const wrapper = shallow(
      <h1>Hello Jest!</h1>
  );
  expect(wrapper).toMatchSnapshot();
});

it('renders correctly', () => {
  const tree = renderer.create(
      <h1>Hello Jest!</h1>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
