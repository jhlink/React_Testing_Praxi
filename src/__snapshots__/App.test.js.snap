// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1>
  Hello Jest!
</h1>
`;

exports[`should render label 1`] = `
<h1>
  Hello Jest!
</h1>
`;
